extension/source/background/KeyringController.ts

```diff
@@ -110,6 +115,19 @@ export default class KeyringController {
       return networkData.address;
     },
 
+    createTempAccount: async (_request) => {
```

Comment:
Same question for `addRecoveryWallet()`, do we need to add unit tests for this new logic? I'm assuming as this is the extension which isn't meant to be production code, we are happy with not adding tests?

Reply:
@JohnGuilding We generally don't have tests for the extension as they would be complicated to setup & manage. However, as extension gets more stable might make sense to add some e2e tests for more common paths. Would save us some regression testing as well.